frontend/javascripts/users/rooms/chat.js

```diff
@@ -141,10 +141,29 @@ export default class Chat {
     }
 
     this.firebase.sendMessage(this.roomId, msgBody, msgType, this.currentUser.id);
+    this.sendNotice();
     $msgField.val('');
     this.resizeTextarea.restoreSize();
   }
 
+  async sendNotice() {
+    var recieverUid;
+    if(this.currentUser.id === this.usersInfo.reviewer.id) {
+      recieverUid = this.usersInfo.reviewee.id;
+    } else {
+      recieverUid = this.usersInfo.reviewer.id;
+    }
+    const recieverPresence = await this.firebase.getPresence(recieverUid);
+    if(recieverPresence.val() && recieverPresence.val().state === 'online') return
+
+    const recieverId = recieverUid.replace('user_', '');
+    const path = '/users/notice';
+    const chatUrl = location.href;
+    const authenticityToken = $('meta[name="csrf_token"]').attr('content');
+    const params = { authenticity_token: authenticityToken, reciever_id: recieverId, chat_url: chatUrl };
```

Comment:
__[Misspell]__

"reciever" is a misspelling of "receiver"


<!-- Comment by [Sider](https://sider.review) -->